devicetypes/ezex/somfy-blind-controller-ezex.src/somfy-blind-controller-ezex.groovy

```diff
@@ -0,0 +1,120 @@
+/**
+ *
+ *	Copyright 2018 eZEX
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+ 
+metadata {
+    definition(name: "SOMFY Blind Controller/eZEX", namespace: "eZEX", author: "eZEX Corp", vid:"SmartThings-smartthings-Somfy_WindowShade", ocfDeviceType: "oic.d.blind") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Window Shade"
+        capability "Health Check"
+        capability "Switch Level"
+        
+        command "pause"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", deviceJoinName: "SOMFY Blind Controller/eZEX", model: "E2B0-KR000Z0-HA"// SY-IoT201-BD
+    }
+
+
+    tiles(scale: 2) {
+         multiAttributeTile(name:"windowShade", type: "generic", width: 6, height: 4){
+            tileAttribute ("device.windowShade", key: "PRIMARY_CONTROL") {
+              attributeState "open", label:'Open', action:"close", icon:"http://www.ezex.co.kr/img/st/window_open.png", backgroundColor:"#00A0DC", nextState:"turningOff"
+              attributeState "close", label:'Closed', action:"open", icon:"http://www.ezex.co.kr/img/st/window_close.png", backgroundColor:"#ffffff", nextState:"turningOn"
+              attributeState "turningOn", label:'Opening',  action:"close", icon:"http://www.ezex.co.kr/img/st/window_open.png", backgroundColor:"#00A0DC", nextState:"turningOff"
+              attributeState "turningOff", label:'Closing', icon:"http://www.ezex.co.kr/img/st/window_close.png", backgroundColor:"#ffffff", nextState:"turningOn"  
```

Comment:
these values should be:

"open", "closed"*, "opening", and "closing"

Reply:
Fixed and will be contained in the next commit.